swift_test.go

```diff
@@ -16,18 +16,19 @@ import (
 	"archive/tar"
 	"bytes"
 	"crypto/md5"
+	"crypto/tls"
 	"encoding/json"
 	"encoding/xml"
 	"fmt"
+	"github.com/ncw/swift"
```

Comment:
goimports puts these lines where they were at the end - I suggest you leave them there otherwise goimports will just put them back!